spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -287,99 +231,22 @@ object external {
-    val sdEntity = hiveStorageDescToEntity(
-      tableDefinition.storage, cluster, db, table
-      /* isTempTable = false  Spark doesn't support temp table */)
-
-    val tblEntity = new AtlasEntity(HIVE_TABLE_TYPE_STRING)
-    tblEntity.setAttribute("qualifiedName",
-      hiveTableUniqueAttribute(cluster, db, table /* , isTemporary = false */))
-    tblEntity.setAttribute("name", table)
-    tblEntity.setAttribute("owner", tableDefinition.owner)
-    tblEntity.setAttribute("ownerType", "USER")
-    tblEntity.setAttribute("tableType", tableDefinition.tableType.name)
-    tblEntity.setAttribute("createTime", new Date(tableDefinition.createTime))
-    tblEntity.setAttribute("parameters", tableDefinition.properties.asJava)
-    tblEntity.setAttribute("schemaDesc", tableDefinition.schema.simpleString)
-    tblEntity.setAttribute("provider", tableDefinition.provider.getOrElse(""))
-    if (tableDefinition.tracksPartitionsInCatalog) {
-      tblEntity.setAttribute("partitionProvider", "Catalog")
-    }
-    tblEntity.setAttribute("partitionColumnNames", tableDefinition.partitionColumnNames.asJava)
-    tableDefinition.comment.foreach(tblEntity.setAttribute("comment", _))
-    tableDefinition.viewText.foreach(tblEntity.setAttribute("viewOriginalText", _))
-
-    tblEntity.setRelationshipAttribute("db", AtlasUtils.entityToReference(dbEntity.entity))
-    tblEntity.setRelationshipAttribute("sd", AtlasUtils.entityToReference(sdEntity.entity))
-
-    new AtlasEntityWithDependencies(tblEntity, Seq(dbEntity, sdEntity))
-  }
-
-  def hiveTableToEntitiesForAlterTable(
-      tblDefinition: CatalogTable,
-      cluster: String,
-      mockDbDefinition: Option[CatalogDatabase] = None): AtlasEntityWithDependencies = {
-    val tableEntity = hiveTableToEntity(tblDefinition, cluster, mockDbDefinition)
-    val deps = tableEntity.dependencies.map(_.entity)
-
-    val dbEntity = deps.filter(e => e.getTypeName.equals(HIVE_DB_TYPE_STRING)).head
-    val sdEntity = deps.filter(e => e.getTypeName.equals(HIVE_STORAGEDESC_TYPE_STRING)).head
-
-    // override attribute with reference - Atlas should already have these entities
-    tableEntity.entity.setRelationshipAttribute("db",
-      AtlasUtils.entityToReference(dbEntity, useGuid = false))
-    tableEntity.entity.setRelationshipAttribute("sd",
-      AtlasUtils.entityToReference(sdEntity, useGuid = false))
-
-    // drop all the dependencies since they're not necessary
-    AtlasEntityWithDependencies(tableEntity.entity)
-  }
-
-  // ================== Hive entities (Hive Warehouse Connector) =====================
-  val HWC_TABLE_TYPE_STRING = "hive_table"
-  val HWC_DB_TYPE_STRING = "hive_db"
-  val HWC_STORAGEDESC_TYPE_STRING = "hive_storagedesc"
-
-  def hwcTableUniqueAttribute(
-      cluster: String,
-      db: String,
-      tableName: String): String = {
-    s"${db.toLowerCase}.${tableName.toLowerCase}@$cluster"
+    hiveTableToReference(db, table, cluster)
   }
 
-  def hwcTableToEntity(
```

Comment:
Now SAC handles Hive table and Hive table for HWC identically, hence no need to maintain.